src/migrations/6-social.ts

```diff
@@ -0,0 +1,56 @@
+import _ from "lodash";
+import { Migration } from "./index.js";
+
+/**
+ * Move `twitter` to `social` and set the name of `social` to the project name.
+ * @param existing
+ * @returns
+ */
+async function updateProjects(existing: any): Promise<any> {
+  if (existing.twitter) {
+    existing.social = {
+      name: existing.name,
+      twitter: `https://twitter.com/${existing.twitter}`
+    };
+    delete existing.twitter;
+  } else {
+    existing.social = {
+      name: existing.name
+    };
+  }
+
+  // Check for Telegram and Discord URLs in websites and move them to social
+  if (existing.websites && Array.isArray(existing.websites)) {
+    const remainingWebsites = [];
+    for (const website of existing.websites) {
+      const url = website.url;
+      if (url && url.startsWith("https://t.me/")) {
```

Comment:
This assumes that there will only be 1 Telegram/Discord/Twitter per project, and any other URLs will get lost (because this is an overwrite. I think it's not uncommon to have multiple Telegram channels that might be related at least, maybe it's worth storing an array of Telegram URLs?

At the very least, we should keep other links in `remainingWebsites` so they don't get lost

Reply:
Make sense, so we can modify social profile in order to use a relationship with url schema and use an array for telegram in order to store multiple channels/bots/groups. 
Something like that?

```json
{
  "$id": "social-profile.json",
  "title": "Social Profile",
  "type": "object",
  "description": "All social profile",
  "properties": {
    "medium": {
      "$ref": "url.json#"
    },
    "twitter": {
      "$ref": "url.json#"
    },
    "telegram": {
      "type": "array",
      "items": {
        "$ref": "url.json#"
      }
    },
    "discord": {
        "$ref": "url.json#"
    },
    "coinmarketcap": {
      "$ref": "url.json#"
    }
  }
}
```